scripts/ci_aws_bootstrap.sh

```diff
@@ -2,76 +2,90 @@
 
 set -euo pipefail
 
-if [[ -z "${RUN_ALL:-}" ]]; then
-    RUN_ALL=0
-fi
+RUN_ALL=${RUN_ALL:-0}
+export VLLM_BUILDKITE_BRANCH="pipeline_gen"
```

Comment:
the branch here would be changed to `main` after merge

Reply:
you can set this to `VLLM_BUILDKITE_BRANCH=${VLLM_BUILDKITE_BRANCH:-main}` and then you can merge it and manually set `VLLM_BUILDKITE_BRANCH=pipeline_gen` or `your-own-branch` when doing testing.